www/js/app.js

```diff
@@ -1132,6 +1186,8 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'cookies','abstractFilesystemAcc
                         selectedArchive.readBinaryFile(dirEntry, function (fileDirEntry, content) {
                             var mimetype = fileDirEntry.getMimetype();
                             // Let's send the content to the ServiceWorker
+                            var message = { 'action': 'giveContent', 'title' : title, 'content': content.buffer, 
+                                'mimetype': mimetype, 'imageDisplay': params.imageDisplayMode };
                             var message = { 'action': 'giveContent', 'title': title, 'content': content.buffer, 'mimetype': mimetype };
```

Comment:
@Hardik500 Why is `message` defined twice here? Clearly defining it the second time will overwrite the first definition.